backend/clubs/models.py

```diff
@@ -698,6 +698,31 @@ def send_approval_email(self, request=None, change=False):
                 reply_to=settings.OSA_EMAILS + [settings.BRANDING_SITE_EMAIL],
             )
 
+    def get_latest_and_latest_approved_versions(self):
+        """
+        Returns the latest and latest approved versions of club key fields
+        [name, description, image] contents that trigger reapproval.
+        """
+        latest_approved_version = (
+            self.history.filter(approved=True).order_by("-history_date").first()
+        )
+        latest_version = self.history.order_by("-history_date").first()
+
+        if (
+            not latest_version
+            or not latest_approved_version
```

Comment:
Like Julian mentioned, this won't work if the club is brand new. We still want to display a diff (I think, I'll defer to @julianweng on this) for clubs that are registering for the first time.

Reply:
I see